package.json

```diff
@@ -120,6 +120,7 @@
     "react-dev-utils": "^6.1.1",
     "react-dom": "^16.6.3",
     "react-element-to-jsx-string": "^14.0.2",
+    "react-helmet-async": "^0.2.0",
```

Comment:
@greg-a-smith does this need to be in regular dependencies or is dev deps ok?

Reply:
Seems like it could just live in dev dependencies.